workflow-job

```diff
@@ -95,7 +95,7 @@ public class CommitStatus implements Serializable {
 }
 
 public class Images implements Serializable {
-    def cassandra = 'cassandra'
+    def cassandra = 'cassandra:2.1.9'
```

Comment:
Updated.
If we say we support cassandra 2.2.3 we should use version 2.2.2.1 of cassandra unit and not 2.1.9.2

See https://github.com/jsevellec/cassandra-unit/wiki/changelog

Do you also want me to contribute this update in James sources ?